packages/cli/src/lib/quire/index.js

```diff
@@ -62,17 +62,19 @@ function getVersion(projectPath) {
  * (i.e `^1.0.0-pre-release.0` => `1.0.0-pre-release.2`) so this string-trimming
  * logic can be removed
  */
-function getVersionFromStarter(projectPath) {
+async function getVersionFromStarter(projectPath) {
   const packageConfig = fs.readFileSync(path.join(projectPath, 'package.json'), { encoding:'utf8' })
   const { peerDependencies } = JSON.parse(packageConfig)
   const version = peerDependencies[PACKAGE_NAME]
-  return version.substr(version.search(/\d/))
+  return version === 'latest'
+    ? await latest()
+    : version.substr(version.search(/\d/))
 }
 
 /**
  * Clone or copy a Quire starter project
  *
- * @param    {String}   starter   A repository URL or path to local starter
+ * @param    {String}   starter   A repository URL or an absolute path to local starter
```

Comment:
**TODO**: resolve both absolute and relative paths to local starter directories in `initStarter`. I updated the JSDoc here since it wasn't clear why this wasn't working at first
```suggestion
 * @param    {String}   starter   A repository URL or an absolute path to local starter
 * @TODO resolve both absolute and relative paths to local starter repositories 
```